diff --git a/node_modules/react-native-gauge/js/Gauge.js b/node_modules/react-native-gauge/js/Gauge.js
index a5021b7..4e8cecc 100644
--- a/node_modules/react-native-gauge/js/Gauge.js
+++ b/node_modules/react-native-gauge/js/Gauge.js
@@ -112,7 +112,7 @@ const Gauge = (props) => {
                     withAnchorPoint(
                       {
                         transform: [
-                          {rotateZ: moveNeedle},
+                          {rotateZ: `${moveNeedle}deg`},
                           {translateY: -translateNeedleY},
                         ],
                       },
@@ -165,7 +165,7 @@ const Gauge = (props) => {
                       width: circleSize,
                       borderRadius: circleSize / 2,
                       backgroundColor: circleColor,
-                      transform: [{rotateZ: moveNeedle}],
+                      transform: [{rotateZ: `${moveNeedle}deg`}],
                     }}
                   />
                 )}
@@ -216,7 +216,7 @@ const Gauge = (props) => {
               withAnchorPoint(
                 {
                   transform: [
-                    {rotateZ: moveNeedle},
+                    {rotateZ: `${moveNeedle}deg`},
                     {translateY: -translateNeedleY},
                   ],
                 },
@@ -252,7 +252,7 @@ const Gauge = (props) => {
                 width: circleSize,
                 borderRadius: circleSize / 2,
                 backgroundColor: circleColor,
-                transform: [{rotateZ: moveNeedle}],
+                transform: [{rotateZ: `${moveNeedle}deg`}],
               }}
             />
           )}
